mergify_engine/worker.py

```diff
@@ -242,6 +242,9 @@ async def _translate_exception_to_retries(
         try:
             yield
         except Exception as e:
+            if isinstance(e, aredis.exceptions.ConnectionError):
+                statsd.increment("engine.redis.connection-errors")
```

Comment:
There's a lot of repetition, we might forget to add this somewhere. Can we wrap the `aredis` client to take care of that?

Reply:
@sileht should I wait for this or you're giving up?

Reply:
Oh, I miss it, but I don't see how to do that properly without involving some python black magic. ConnectionError can be raised everywhere in aredis code.